ietf/group/admin.py

```diff
@@ -327,11 +327,19 @@ def value_from_datadict(self, data, files, name):
             bits = upload.read()
             return b64encode(bits).decode("ascii") # Who made this so hard?
             
+class RestrictContentTypeChoicesForm(forms.ModelForm):
+    content_type = forms.ChoiceField(
+        choices=(
+            ( "text/markdown;charset=utf-8", "Markdown"),
+            ( "application/pdf;charset=utf-8", "PDF")
```

Comment:
As noted elsewhere with references, I don't think `charset` is valid for `application/pdf`